### DISCUSSION TITLE
Taking a break for some days

### DISCUSSION THREAD

**Discussion by stax76** [staxrip/staxrip#550] [General]
Hi everybody,

I will have 9 days rest until the end of next week. Usually I spent way too much time sitting in front of my PC, my plan is to avoid any PC activities altogether in that time and generally taking care of my health.

See you then. Have a good time. 😊

**Comment by JJKylee** [comment]
Take good care, and see ya. 😉

**Reply by stax76** [reply]
Thanks 😀

**Comment by Dendraspis** [comment]
Sounds like what I should do, too!  😊

But I wished you would start tomorrow to release my next big commit. 😂

Wish you a great time ... gute Besserung!

**Reply by ghost** [reply]
Well, indeed, enjoy your time !!!

**Reply by Dendraspis** [reply]
First I have to finish the highlighing thing.  😁

**Comment by Patman86** [comment]
You have really given full throttle here in the last few weeks, so the break is well deserved.  Don't forget that everything is a hobby!  Health and family (maybe also the job) always come first!
@stax76 Get well soon and enjoy the break!

**Comment by Dendraspis** [comment]
# Welcome back!  🎉  🥳

**Reply by ghost** [reply]
Haha indeed !!! welcome back to Stax76! I've just saw the commit 😁

**Reply by stax76** [reply]
Thank you guys, it was a good experience. I had used the PC only for media search and WhatsApp.

**Reply by ghost** [reply]
Well maybe next time, you'll even not use whatsapp, only media search 🤣and you'll probably rest even better 😁

**Reply by Dendraspis** [reply]
...at least faster.   😁

**Comment by stax76** [comment]
Now that Dendraspis is doing a fantastic job on maintenance tasks (meaning project lead) and I want to work on mpv.net, it makes sense for me planing a bit.

Letting it go is not easy, and I still like working on staxrip and like being around here with you guys, I think I will try some kind of transition until mpv.net maintenance gains traction.

I think I will try a fixed time, like working on staxrip the first week (or maybe two in the beginning) of each month. So will be around for a while and work on some items I've interest, tabcontrol, datagrid, docs, small features and support.

**Reply by Dendraspis** [reply]
> Letting it go is not easy

Really? We didn't notice that.  😁  😂  😉 

Sounds like a plan.  😄   And nobody says you have to go - just take your break and come back whenever you want.  🙂   You're still the one who keeps us together.  😉

**Reply by ghost** [reply]
Well I wish you, as always, a good resting time @stax76 ! ❤️ 
And I'm looking forward to your return, but I hope you'll be more careful with synchronizing your efforts with @Dendraspis  next time you're back. It is very important to work together and not everyone at his own side.
That is of course, only my humble opinion...

**Reply by JJKylee** [reply]
> So will be around for a while and work on some items I've interest, tabcontrol, datagrid, docs, small features and support.

Yeah, the final remaining items that are still not themed.
Good to hear that. 😁

And let's try to stay not stressed, y'all~ 😅
